src/styles/codeStyles/index.ios.js

```diff
@@ -0,0 +1,16 @@
+const codeWordWrapper = {
+    height: 18,
+    justifyContent: 'center',
+    marginTop: 2,
+};
+
+const codeWordStyle = {
+    height: 16,
+    top: 4,
+};
+
+const codeTextStyle = {
+    lineHeight: 15,
+};
```

Comment:
The line height here was not enough and caused a regression https://github.com/Expensify/App/issues/19922 when the inline code contains emojis.